test/server/cards/02_SHD/units/SupercommandoSquad.spec.js

```diff
@@ -0,0 +1,31 @@
+describe('Supercommando Squad', function() {
+    integration(function() {
+        describe('Supercommando Squad\' ability', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: [{ card: 'supercommando-squad', upgrades: ['shield'] }],
+                    },
+                    player2: {
+                        groundArena: ['wampa', 'jedha-agitator'],
+                    }
+                });
+            });
+
+            it('should give it sentinel only as long as it is upgraded', function () {
+                this.player1.pass();
+                this.player2.clickCard(this.wampa);
+                //Supercommando Squad automatically selected due to sentinel
+                expect(this.player1).toBeActivePlayer();
+                //no damage because of shield
```

Comment:
Can we add a couple more expectations just to confirm combat resolved the way we think it did:
- supercommando squad shield was defeated - `expect(this.supercommandoSquad.upgraded()).toBe(false)`
- wampa has 4 damage